new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSystemResOverrides.java

```diff
@@ -0,0 +1,514 @@
+  final String wlSecretName = "weblogic-credentials";
+  final String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+  int replicaCount = 2;
+
+  static Path sitconfigAppPath;
+  String overridecm = "configoverride-cm";
+  LinkedHashMap<String, DateTime> podTimestamps;
+
+  // create standard, reusable retry/backoff policy
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(5, MINUTES).await();
+  private static LoggingFacade logger = null;
+
+  /**
+   * Assigns unique namespaces for operator and domains.
+   * Pulls WebLogic image if running tests in Kind cluster.
+   * Installs operator.
+   * Creates and starts WebLogic domain containing 2 instances in dynamic cluster.
+   * Creates JMS and WLDF system resources.
+   * Deploys sitconfig application to cluster and admin targets.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+
+    logger.info("Assign a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    opNamespace = namespaces.get(0);
+    logger.info("Assign a unique namespace for domain namspace");
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    domainNamespace = namespaces.get(1);
+
+    // install operator and verify its running in ready state
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    //determine if the tests are running in Kind cluster. if true use images from Kind registry
+    if (KIND_REPO != null) {
+      String kindRepoImage = KIND_REPO + image.substring(TestConstants.OCR_REGISTRY.length() + 1);
+      logger.info("Using image {0}", kindRepoImage);
+      image = kindRepoImage;
+      isUseSecret = false;
+    } else {
+      // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
+      createOCRRepoSecret(domainNamespace);
+    }
+    //create and start WebLogic domain
+    createDomain();
+
+    // build the sitconfig application
+    Path distDir = buildApplication(Paths.get(APP_DIR, "sitconfig"),
+        null, null, "dist", domainNamespace);
+    sitconfigAppPath = Paths.get(distDir.toString(), "sitconfig.war");
+    assertTrue(sitconfigAppPath.toFile().exists(), "Application archive is not available");
+
+    //deploy application to view server configuration
+    deployApplication(clusterName + "," + adminServerName);
+
+  }
+
+  /**
+   * Test JMS and WLDF system resources configurations are overridden dynamically when domain resource
+   * is updated with overridesConfigMap property. After the override verifies the system resources properties
+   * are overridden as per the values set.
+   */
+  @Test
+  @DisplayName("Test JMS and WLDF system resources override")
+  public void testSystemResourceOverride() {
```

Comment:
Modify Test name to "testJmsWldfSystemResourceOverride" to describe the usecase.

Reply:
changed